roles/pulp_common/defaults/main.yml

```diff
@@ -22,7 +22,8 @@ pulp_group_id:
 pulp_user_home: '/var/lib/pulp'
 pulp_pip_editable: yes
 pulp_use_system_wide_pkgs: false
-prereq_pip_packages: []
+prereq_pip_packages:
+  - Jinja2
```

Comment:
```
Aug 18 22:22:07 debian-10 systemd[1]: Stopped Pulp RQ Worker.
Aug 18 22:22:07 debian-10 systemd[1]: pulpcore-worker@1.service: Consumed 4.126s CPU time.
Aug 18 22:22:07 debian-10 systemd[1]: Started Pulp RQ Worker.
Aug 18 22:22:07 debian-10 sudo[9220]: pam_unix(sudo:session): session closed for user root
Aug 18 22:22:08 debian-10 gunicorn[9167]: [2020-08-18 22:22:08 +0000] [9173] [ERROR] Exception in worker process
Aug 18 22:22:08 debian-10 gunicorn[9167]: Traceback (most recent call last):
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "/usr/local/lib/pulp/lib/python3.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
Aug 18 22:22:08 debian-10 gunicorn[9167]:     worker.init_process()
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "/usr/local/lib/pulp/lib/python3.7/site-packages/aiohttp/worker.py", line 52, in init_process
Aug 18 22:22:08 debian-10 gunicorn[9167]:     super().init_process()
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "/usr/local/lib/pulp/lib/python3.7/site-packages/gunicorn/workers/base.py", line 119, in init_process
Aug 18 22:22:08 debian-10 gunicorn[9167]:     self.load_wsgi()
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "/usr/local/lib/pulp/lib/python3.7/site-packages/gunicorn/workers/base.py", line 144, in load_wsgi
Aug 18 22:22:08 debian-10 gunicorn[9167]:     self.wsgi = self.app.wsgi()
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "/usr/local/lib/pulp/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
Aug 18 22:22:08 debian-10 gunicorn[9167]:     self.callable = self.load()
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "/usr/local/lib/pulp/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load
Aug 18 22:22:08 debian-10 gunicorn[9167]:     return self.load_wsgiapp()
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "/usr/local/lib/pulp/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 39, in load_wsgiapp
Aug 18 22:22:08 debian-10 gunicorn[9167]:     return util.import_app(self.app_uri)
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "/usr/local/lib/pulp/lib/python3.7/site-packages/gunicorn/util.py", line 358, in import_app
Aug 18 22:22:08 debian-10 gunicorn[9167]:     mod = importlib.import_module(module)
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
Aug 18 22:22:08 debian-10 gunicorn[9167]:     return _bootstrap._gcd_import(name[level:], package, level)
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "/usr/local/lib/pulp/lib/python3.7/site-packages/pulpcore/content/__init__.py", line 21, in <module>
Aug 18 22:22:08 debian-10 gunicorn[9167]:     from .handler import Handler  # noqa: E402: module level not at top of file
Aug 18 22:22:08 debian-10 gunicorn[9167]:   File "/usr/local/lib/pulp/lib/python3.7/site-packages/pulpcore/content/handler.py", line 33, in <module>
Aug 18 22:22:08 debian-10 gunicorn[9167]:     from jinja2 import Template  # noqa: E402: module level not at top of file
Aug 18 22:22:08 debian-10 gunicorn[9167]: ModuleNotFoundError: No module named 'jinja2'
```

Reply:
If pulpcore content handler imports `jinja2` it must be in its `requirements.txt` or am i missing something here?

Reply:
Yeah, and even if we were supposed to install it in the installer, it should be a separate install task. prereq_pip_packages is a hook for the prereq roles to add packages after the venv is created (it is created after the prereq roles are run.)

Reply:
as the previous versions did not have `jinja2` as a requirement, I will keep it here for now.
I opened a PR on pulpcore: https://github.com/pulp/pulpcore/pull/857